### ISSUE TITLE
Incorrect use of lcg-info

### ISSUE THREAD

**Issue by ericvaandering** [dmwm/CRAB2#266]
Original Savannah ticket 45847 reported by gcodispo on Tue Jan 13 05:31:34 2009.

I've tried to send a task with CRAB to PIC with the /cms/Role=production FQAN, but no CE is selected by the query

lcg-info --vo cms --list-ce --query 'Tag=VO-cms-CMSSW_2_1_7,Tag=VO-cms-slc4_ia32_gcc345,CEStatus=Production,CloseSE=srmcms.pic.es' --sed

because now the CEs at PIC only support /cms/Role=lcgadmin and /cms/Role=production and not the whole cms VO. The correct command would be

lcg-info --vo VOMS:/cms/Role=production --list-ce --query 'Tag=VO-cms-CMSSW_2_1_7,Tag=VO-cms-slc4_ia32_gcc345,CEStatus=Production,CloseSE=srmcms.pic.es' --sed

In other words, if the primary FQAN in the proxy is not /cms, lcg-info should be called twice: one with the primary FQAN and one with the VO, and the outputs should be merged.

As a consequence of this bug, is currently impossible to send Job Robot jobs to PIC.

**Comment by ericvaandering** [comment]
Closed by gcodispo on Tue Jan 20 13:17:36 2009
